app/experimenter/nimbus-ui/src/components/TableHighlights/index.tsx

```diff
@@ -29,7 +29,7 @@ const getHighlightMetrics = (
   const highlightMetricsList = [...HIGHLIGHTS_METRICS_LIST];
   probeSets.forEach((probeSet) => {
     highlightMetricsList.unshift({
-      value: `${probeSet!.slug.replace(/-/g, "_")}_ever_used`,
+      value: `${probeSet!.slug}_ever_used`,
```

Comment:
@emtwo I'll double check this in the morning, but looks like the slugs for probe sets are already snake cased. Otherwise, I'll change this in all 4 tables.

Reply:
@jaredlockhart is working on the Metrics page - it seems like I could add `.replace(/-/g, "_")` in the four places we need `_ever_used` as a safeguard either way, but I want to make sure our mocked data reflects what we expect to see in production so I'll wait for his input. 👀  So Jared, `🐍 || 🍡`?

Reply:
@emtwo Jared got back to me, says we should file a ticket for enforcing snake-case on probe set slugs since Jetstream uses snake case. So, this should be g2g.

Reply:
awesome! thanks